examples/trigonometry.R

```diff
@@ -0,0 +1,71 @@
+# https://processing.org/tutorials/trig/
+
+px <- 0
+py <- 0
+px2 <- 0
+py2 <- 0
+angle <- 0
+angle2 <- 0
+x <- 0
+x2 <- 0
+radius <- 50
+frequency <- 2
+frequency2 <- 2
```

Comment:
Brainstorming: In the future when choosing R style for the example code we might consider grouping variables for global declarations. Shortening these long variable stacks at the top would make simple examples less imposing.

Does R have a common style for multiple variable assignment on a single line?, the equivalent of:

```
int x, y;
int a =0, b = 0;
int n = m = 0;
```

Perhaps still readable but more concise would be grouping a few like terms together on each line, something like (untested):

```
px <- 0; py <- 0
px2 <- 0; py2 <- 0
angle <-0; angle2 <- 0
x <- 0; x2 <- 0
```

Or something like (untested):

`frequency <- frequency2 <- 2`